### ISSUE TITLE
是否已经支持elasticsearch 8.0？

### ISSUE THREAD

**Issue by mengguiyouziyi** [mobz/elasticsearch-head#475]

**Comment by linghengqian** [comment]
You can experiment, but I'm not optimistic about it.

**Comment by holdlg** [comment]
The connection ElasticSearch 8 source field is not displayed
![image](https://user-images.githubusercontent.com/4408716/163553409-1ec41d96-0a56-45c4-9633-e11c3de161ce.png)

**Comment by xiaodizi** [comment]
No, not supported, ES 8.1 is OK, ES 8.6 is not OK. I tried that
